### ISSUE TITLE
feat: Allow for no-config adds

### ISSUE THREAD

**Issue by ieedan** [ieedan/jsrepo#112]
This would allow you to add without a config file. Much nicer than erroring without a config file. Tests and diffs should still require one though.

Basically in the `add` command it would prompt you:

```
1. Would you like to add a config? Y/n (recommended)
-- yes path --
run `init`
-- no path --
wait until after blocks are selected
where would you like to add the blocks? 
src/blocks
would you like to include tests? y/N
```

I think this would be nice if a library wanted to just tell their users to install their components by saying:
```bash
npx jsrepo add github/ieedan/std/utilities/math
```

**Comment by ieedan** [comment]
Making a change to this instead of prompting for config we are just going to recommend you run `init` if there is no config and the `--no-config` flag is not true.

**Comment by ieedan** [comment]
On second though actually implementing this...

This really has no utility in this project. You lose the ability to run `diff` and `test` and really for what?

**Comment by ieedan** [comment]
I still think the utility in this is there I think a "continue without config" might be nice but then asking you to supply a path for install.

**Comment by ieedan** [comment]
I would also like to add this is pretty low priority at the moment I have some other things that I think should be added first.

**Comment by ieedan** [comment]
A little better planning for this one...

I think users not using a config should be prompted by the cli when running `add` along the lines of:

```
You don't have jsrepo initialized in your project. Do you want to continue? 
y/N

after: After the user has chosen the blocks they want to install

if: Y
Where would you like to install the blocks?
<path>

if: any of the blocks have tests
Include tests?
y/N

Add a watermark?
Y/n
```
